src/test/java/core/basesyntax/FilterOfNumberTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class FilterOfNumberTest {
+    @Test(expected = ArithmeticException.class)
+    public void checkDivisionRemainder() {
+        Random random = new Random();
```

Comment:
it's a bad practice to use random data in tests. If there is some edge case which is failing, test starts failing once in 20-50 runs and it's really hard to find a pattern. Those tests are called flaky. Flaky tests are worse than no tests, because you stop trusting your tests. Let's hardcode some data instead of generating random data.